tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -64,15 +64,16 @@ public function testBucketDetail()
         $this->assertEquals($tokenData['owner']['defaultBackend'], $bucket['backend']);
         $this->assertNotEquals($displayName, $bucket['displayName']);
 
-        $bucketUpdateOptions = new BucketUpdateOptions($bucketId, $displayName);
-        $bucket = $this->_client->updateBucket($bucketUpdateOptions);
+        $bucketUpdateOptions = new BucketUpdateOptions($bucketId, $displayName, true);
```

Comment:
Tím, že jsi to jen přepsal a nepřidal zmizel test na synchronní update bucketu.

Reply:
Dik, ja som mal dojem ze som kontroloval ze sa to este niekde testuje, vratil som tam ten test a pridal testovanie toho async